packages/orbit-components/src/utils/Grid/types.d.ts

```diff
@@ -18,6 +18,7 @@ export interface BasicProps extends Common.SpaceAfter {
 
 export interface Props extends Common.Globals, Common.SpaceAfter, BasicProps {
   readonly as?: string;
+  readonly className?: string;
```

Comment:
separeted feat commit 🙏🏻 and do not forget also to add that to flow file ;)

Reply:
done 👌